### PR TITLE
Add deliverer token related e2e cases to puzzle_delivery.feature

### PR DESCRIPTION
**Pull request by dannyh79**
#11

As title ([reference](https://github.com/CCIP-App/CCIP-Server/blob/5175429a0418fde9cfe636ee1107001a17343215/app/ccip.py#L272-L280)). CCIP-server's [`deliver_puzzle()` and logging behavior](https://github.com/CCIP-App/CCIP-Server/blob/5175429a0418fde9cfe636ee1107001a17343215/app/ccip.py#L273-L274) TBD.

### CODE REVIEW

File: api/schema/puzzle.ts [hunk 1]
```diff
@@ -9,3 +9,11 @@ export type PuzzleStatus = {
 export type DeliverPuzzlePayload = {
   receiver: string
 }
+
+export type PuzzleDeliveredResponse = {
+  status: 'OK'
+  /**
```
Comment (elct9620) [inline api/schema/puzzle.ts:15]: 這裡為什麼要註解？

Comment (dannyh79) [reply api/schema/puzzle.ts:15]: 認為單看 user_id 無法辨別此為 deliverer ID 或是 receiver ID ，故提供此可供 tsserver 顯示的註解：
image

Comment (elct9620) [reply api/schema/puzzle.ts:15]: 用 TSDoc 的格式寫
https://tsdoc.org/

Comment (dannyh79) [reply api/schema/puzzle.ts:15]: 83c2bf6

**Pull request merged**